blog/views.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+from django.shortcuts import get_object_or_404, render, redirect
+from django.views import generic
+
+from blog.forms import CommentForm
+from blog.models import Post
+
+
+class PostListViews(generic.ListView):
+    model = Post
+    template_name = "index.html"
+    paginate_by = 5
+
+
+class PostDetailView(generic.View):
+    template_name = "blog/post_detail.html"
+    queryset = Post.objects.prefetch_related("comments__user")
+
+    def get(self, request, pk):
+        post = get_object_or_404(Post, pk=pk)
+        comment_form = CommentForm()
+        context = {
+            "post": post,
+            "comments": post.comments,
+            "comment_form": comment_form,
+            "user": request.user
+        }
+
+        return render(request, self.template_name, context)
+
+    def post(self, request, pk):
+        post = get_object_or_404(Post, pk=pk)
+        comment_form = CommentForm(request.POST)
+
+        if comment_form.is_valid():
+            new_comment = comment_form.save(commit=False)
+            content = comment_form.cleaned_data.get("content")
+            new_comment.content = content
```

Comment:
Can you try without this line? I think this line also redundant

Reply:
After this form do not work.